src/main/webapp/user_profile.html

```diff
@@ -28,19 +28,36 @@ <h1 id="title">My Tasks</h1>
     </div>
     <br/>
     <div id="task-list">
-        <table id="need-help"></table>
-        <table id="offer-help"></table>
+        <table class="task-table" id="need-help">
+            <tr>
```

Comment:
Header rows should belong to a `thead` element. You can also pre-populate this table with an empty `tbody` element for the data rows to be added.

Reply:
Sure. In the new commit the header row is put in a `thead` element and each time I load the table, I first empty the `tbody` element. Then appending `tr` element to `tbody` one by one.